en-US/Language.xml

```diff
@@ -946,6 +946,50 @@ Numbers (move from N entry) -->
 		</para>
 	</section>
 
+<!-- Added section for redundant words -->
+	 <section id="redundant">
+		<title>Avoiding Redundant Words</title>
+		 <para>
+			Avoid redundant words, such as "actually", "really", "simply" and "very". They are typically filler words that add no value to a sentence.  
+		</para>
+					 <table>
+						<title></title>
+						 <tgroup cols="2" colsep="1" rowsep="1">
+							<colspec colname="c1"></colspec>
+							 <colspec colname="c2"></colspec>
+							 <thead>
+								<row>
+									<entry> Example </entry>
+									 <entry> Improvement </entry>
+
+								</row>
+
+							</thead>
+							 <tbody>
+								<row>
+									<entry> Remote users can connect to network resources simply by authenticating to their local machine. </entry>
+									 <entry> Remote users can connect to network resources by authenticating to their local machine. </entry>
+
+								</row>
+								<row>
+									<entry> Any settings that are changed with this command revert back to their original settings when the daemon restarts. </entry>
```

Comment:
I'm not so sure about these examples. The section intro talks about filler words, but not about other kinds of redundant words. 
It took me a minute to find the difference and the words changed They aren't wrong examples, obviously, but maybe not the most impactful. 
Perhaps add slightly more to the intro, indicating that redundant words might also include examples where two words or phrases are used that mean approximately the same thing (revert back) or that add nothing to the sentence (on the partition, vs. located on the partition)

Reply:
Added more to the introductory text to clarify these distinctions.